README.rst

```diff
@@ -1,7 +1,7 @@
 Legacy NS1 Python SDK
 =====================
 
-This SDK is deprecated in favor of `ns1-python <https://github.com/ns1/ns1-python>`_
+This SDK is [deprecated](https://github.com/ns1/community/blob/master/project_status/DEPRECATED.md) in favor of `ns1-python <https://github.com/ns1/ns1-python>`_
 and should not be used for new projects. The code in this repository provides
 just a thin wrapper for the new SDK to allow easier migration.
```

Comment:
can we split out the first sentence and use the blockquote thing so it's consistent? I would keep the wording in the 1st sentence as you have it, and the 2nd sentence isn't part of the status blurb